bridges/primitives/chain-bridge-hub-cumulus/src/lib.rs

```diff
@@ -18,7 +18,7 @@
 
 pub use bp_polkadot_core::{
 	AccountId, AccountInfoStorageMapKeyProvider, AccountPublic, Balance, Block, BlockNumber, Hash,
-	Hasher, Hashing, Header, Index, Nonce, Perbill, Signature, SignedBlock, UncheckedExtrinsic,
+	Hasher, Hashing, Header, Nonce, Perbill, Signature, SignedBlock, UncheckedExtrinsic,
```

Comment:
What was the `Nonce` that this was already using?

Reply:
It wasn't used anywhere in here, and in the definition it was the same as `Index`

https://github.com/paritytech/cumulus/blob/2ec939221c4afb287a37373be04bd29b8f5a1d60/bridges/primitives/polkadot-core/src/lib.rs#L184-L209